### ISSUE TITLE
 UNEXPECTED TOP-LEVEL EXCEPTION

### ISSUE THREAD

**Issue by VinothiniBalakrishnan** [rollbar/rollbar-android#15]
Hi 
I am trying to integrate Rollbar with my Android app. My Android app was build in Cordova with Angualr Js. I followed Android instructions present at https://rollbar.com/vinothinibm/FirstProject/#android. I downloaded  rollbar-android.jar  when I make Android build in cordova I can see MyApp\platforms\android\libs 
I placed rollbar-android.jar under libs directory and run
"Cordova build android"
then got 

```
-dex:
      [dex] input: C:\Vinothini\Projects\Hippocratus\hybridapp\platforms\android\ant-build\classes
      [dex] input: C:\Vinothini\Projects\Hippocratus\hybridapp\platforms\android\CordovaLib\ant-build\classes.jar
      [dex] input: C:\Vinothini\Projects\Hippocratus\hybridapp\platforms\android\libs\rollbar-android-0.1.1.jar
      [dex] Using Pre-Dexed classes-b23aab4ccf5bfd309339368db6bdc743.jar <- C:\Vinothini\Projects\Hippocratus\hybridapp\platforms\android\CordovaLib\ant-bu
ild\classes.jar
      [dex] Pre-Dexing C:\Vinothini\Projects\Hippocratus\hybridapp\platforms\android\libs\rollbar-android-0.1.1.jar -> rollbar-android-0.1.1-4498509fa50c3f
f2998fe3f4e506042a.jar
       [dx]
       [dx] UNEXPECTED TOP-LEVEL EXCEPTION:
       [dx] com.android.dx.cf.iface.ParseException: bad class file magic (cafebabe) or version (0034.0000)
       [dx]     at com.android.dx.cf.direct.DirectClassFile.parse0(DirectClassFile.java:472)
       [dx]     at com.android.dx.cf.direct.DirectClassFile.parse(DirectClassFile.java:406)
       [dx]     at com.android.dx.cf.direct.DirectClassFile.parseToInterfacesIfNecessary(DirectClassFile.java:388)
       [dx]     at com.android.dx.cf.direct.DirectClassFile.getMagic(DirectClassFile.java:251)
       [dx]     at com.android.dx.command.dexer.Main.processClass(Main.java:665)
       [dx]     at com.android.dx.command.dexer.Main.processFileBytes(Main.java:634)
       [dx]     at com.android.dx.command.dexer.Main.access$600(Main.java:78)
       [dx]     at com.android.dx.command.dexer.Main$1.processFileBytes(Main.java:572)
       [dx]     at com.android.dx.cf.direct.ClassPathOpener.processArchive(ClassPathOpener.java:284)
       [dx]     at com.android.dx.cf.direct.ClassPathOpener.processOne(ClassPathOpener.java:166)
       [dx]     at com.android.dx.cf.direct.ClassPathOpener.process(ClassPathOpener.java:144)
       [dx]     at com.android.dx.command.dexer.Main.processOne(Main.java:596)
       [dx]     at com.android.dx.command.dexer.Main.processAllFiles(Main.java:498)
       [dx]     at com.android.dx.command.dexer.Main.runMonoDex(Main.java:264)
       [dx]     at com.android.dx.command.dexer.Main.run(Main.java:230)
       [dx]     at com.android.dx.command.dexer.Main.main(Main.java:199)
       [dx]     at com.android.dx.command.Main.main(Main.java:103)
       [dx] ...while parsing com/rollbar/android/Notifier$1.class
       [dx]
       [dx] 1 error; aborting

BUILD FAILED
C:\Vinothini\Setup\Android\adt-bundle-windows-x86_64-20140321\adt-bundle-windows-x86_64-20140321\sdk\tools\ant\build.xml:892: The following error occurred
while executing this line:
C:\Vinothini\Setup\Android\adt-bundle-windows-x86_64-20140321\adt-bundle-windows-x86_64-20140321\sdk\tools\ant\build.xml:894: The following error occurred
while executing this line:
C:\Vinothini\Setup\Android\adt-bundle-windows-x86_64-20140321\adt-bundle-windows-x86_64-20140321\sdk\tools\ant\build.xml:906: The following error occurred
while executing this line:
C:\Vinothini\Setup\Android\adt-bundle-windows-x86_64-20140321\adt-bundle-windows-x86_64-20140321\sdk\tools\ant\build.xml:284: null returned: 1

Total time: 3 seconds

C:\Vinothini\Projects\Hippocratus\hybridapp\platforms\android\cordova\node_modules\q\q.js:126
                    throw e;
                          ^
Error code 1 for command: cmd with args: /s,/c,ant,debug,-f,C:\Vinothini\Projects\Hippocratus\hybridapp\platforms\android\build.xml,-Dout.dir=ant-build,-Dg
en.absolute.dir=ant-gen
Error: C:\Vinothini\Projects\Hippocratus\hybridapp\platforms\android\cordova\build.bat: Command failed with exit code 8
    at ChildProcess.whenDone (C:\Users\vinothini\AppData\Roaming\npm\node_modules\cordova\node_modules\cordova-lib\src\cordova\superspawn.js:135:23)
    at ChildProcess.emit (events.js:98:17)
    at maybeClose (child_process.js:756:16)
    at Process.ChildProcess._handle.onexit (child_process.js:823:5)
```

Any idea?
I included https://raw.githubusercontent.com/rollbar/rollbar.js/master/release/rollbar-0.10.10.js in my app

AndroidManifest.xml contain ```<uses-permission android:name="android.permission.INTERNET" />```

**Comment by powder366** [comment]
I also have this. Was to fast and entered an new issue 16. Sorry.

**Comment by coryvirok** [comment]
Fixed with PR#14
